events/triggers/update

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+UMBREL_ROOT=$(dirname $(readlink -f ../../$0))
+# UMBREL_ROOT=/home/umbrel
+RELEASE="v$(cat $UMBREL_ROOT/events/signals/update)"
+UMBREL_USER=umbrel
+
+if [ $(whoami) != 'root' ]; then
+    UMBREL_USER=$(whoami)
+fi
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "========== Stage: Download ============"
+echo "======================================="
+echo
+
+if [ -z $(grep '[^[:space:]]' $UMBREL_ROOT/events/signals/update) ]; then
+    echo "Empty update signal file. No release version not found."
+    exit 1
+fi
+
+# Make sure an update is not in progres
+if [ -f "$UMBREL_ROOT/statuses/update-in-progress" ]; then 
+    echo "An update is already in progress. Exiting now."
+    exit 2
+fi
+
+echo "Creating lock"
+touch $UMBREL_ROOT/statuses/update-in-progress
+
+# Cleanup just in case there's temp stuff lying around from previous update
+echo "Cleaning up any previous mess"
+[ -d /tmp/umbrel-$RELEASE ] && rm -rf /tmp/umbrel-$RELEASE
+
+# Update status file
+cat <<EOF > $UMBREL_ROOT/statuses/update-status.json
+{"state": "installing", "progress": 10, "description": "Downloading Umbrel $RELEASE"}
+EOF
+
+# Clone new release
+echo "Downloading Umbrel $RELEASE"
+
+cd /tmp/umbrel-$RELEASE
+wget -qO- "https://raw.githubusercontent.com/mayankchhabra/umbrel/$RELEASE/install-box.sh" | sh
```

Comment:
And we should do the same for any other assets that are downloaded by `install-box.sh`.

I see `install-box.sh` does a git clone. It might be better to instead pull the release tarball and publish a signed shasum for the release assets so everything can be verified in one go.

Reply:
That's a good idea regarding release verification. Let's patch both `install-box.sh` and the `update` trigger as soon as we set up our keybase with a new PGP key.

Reply:
Do you want me to get this setup now?

Reply:
Ah I need to setup `MX` for `getumbrel.com` and create an email address first. Or can you add the email to an existing PGP key's meta-data later?

Reply:
You don't need to prove anything to add it to the key. I can set it to `pgp@getumbrel.com` and we can create it in the future.

Reply:
Perfect, that sounds good. Let's run with it then!